Index: configure.ac
===================================================================
RCS file: /usr/local/cvsroot/slony1/slony1-engine/configure.ac,v
retrieving revision 1.47
diff -c -r1.47 configure.ac
*** configure.ac	31 May 2005 04:37:20 -0000	1.47
--- configure.ac	2 Jun 2005 12:02:02 -0000
***************
*** 37,43 ****
      irix*) template=irix ;;
     linux*|gnu*|k*bsd*-gnu) 
             template=linux ;;
!    mingw*) template=win ;;
    netbsd*) template=netbsd ;;
  nextstep*) template=nextstep ;;
   openbsd*) template=openbsd ;;
--- 37,43 ----
      irix*) template=irix ;;
     linux*|gnu*|k*bsd*-gnu) 
             template=linux ;;
!    mingw*) template=win32 ;;
    netbsd*) template=netbsd ;;
  nextstep*) template=nextstep ;;
   openbsd*) template=openbsd ;;
Index: config/libc.m4
===================================================================
RCS file: /usr/local/cvsroot/slony1/slony1-engine/config/libc.m4,v
retrieving revision 1.3
diff -c -r1.3 libc.m4
*** config/libc.m4	2 Apr 2005 08:25:43 -0000	1.3
--- config/libc.m4	2 Jun 2005 12:02:03 -0000
***************
*** 12,21 ****
      HAVE_POSIX_SIGNALS=$slonac_cv_func_posix_signals  
    else
      if test x"$template" = "xwin" ; then
!       AC_MSG_RESULT("Skipping Error on win32")
!     else
!       AC_MSG_RESULT("error")
!       AC_MSG_ERROR(Slony requires a POSIX compatible signal interface.)
      fi
    fi
    AC_SUBST(HAVE_POSIX_SIGNALS)]
--- 12,24 ----
      HAVE_POSIX_SIGNALS=$slonac_cv_func_posix_signals  
    else
      if test x"$template" = "xwin" ; then
!       AC_MSG_RESULT(*** Skipping error on Cygwin ***)
!     else 
!       if test x"$template" = "xwin32" ; then
!         AC_MSG_RESULT(*** Skipping error on MinGW/Msys to allow xxid/slony_funcs compilation ***)
!       else
!         AC_MSG_ERROR(Slony requires a POSIX compatible signal interface.)
!       fi
      fi
    fi
    AC_SUBST(HAVE_POSIX_SIGNALS)]
Index: makefiles/Makefile.win32
===================================================================
RCS file: /usr/local/cvsroot/slony1/slony1-engine/makefiles/Makefile.win32,v
retrieving revision 1.1
diff -c -r1.1 Makefile.win32
*** makefiles/Makefile.win32	9 Jul 2004 07:29:46 -0000	1.1
--- makefiles/Makefile.win32	2 Jun 2005 12:02:03 -0000
***************
*** 1,12 ****
! # $PostgreSQL: pgsql-server/src/makefiles/Makefile.win32,v 1.3 2003/11/29 19:52:12 pgsql Exp $
  
  # Use replacement include files for those missing on Win32
! override CPPFLAGS+="-I$(top_srcdir)/src/include/port/win32"
! 
  DLLTOOL= dlltool
  DLLWRAP= dllwrap
! BE_DLLLIBS= -L$(top_builddir)/src/backend -lpostgres
! DLLINIT = $(top_builddir)/src/utils/dllinit.o
  
  AROPT = crs
  DLSUFFIX = .dll
--- 1,11 ----
! # $Header: /local/home/ssinger/cvs2svn/cvs2svn-2.3.0/slony-cvsd/slony1-engine/Win32/Attic/win32_build.diff,v 1.1 2005-06-02 18:35:40 cbbrowne Exp $
  
  # Use replacement include files for those missing on Win32
! override CPPFLAGS+=-I${pgincludeserverdir}/port/win32
! 
  DLLTOOL= dlltool
  DLLWRAP= dllwrap
! DLLINIT = ${pglibdir}/pgxs/src/utils/dllinit.o -L${pglibdir} -lpostgres
  
  AROPT = crs
  DLSUFFIX = .dll
***************
*** 16,33 ****
  	$(DLLTOOL) --export-all --output-def $*.def $<
  	$(DLLWRAP) -o $@ --def $*.def $< $(DLLINIT) $(SHLIB_LINK)
  	rm -f $*.def
- 
- ifneq (,$(findstring backend,$(subdir)))
- ifeq (,$(findstring conversion_procs,$(subdir)))
- override CPPFLAGS+= -DBUILDING_DLL
- endif
- endif
- 
- ifneq (,$(findstring ecpg/lib,$(subdir)))
- override CPPFLAGS+= -DBUILDING_DLL
- endif
- 
- # required by Python headers
- ifneq (,$(findstring src/pl/plpython,$(subdir)))
- override CPPFLAGS+= -DUSE_DL_IMPORT
- endif
--- 15,17 ----
